revscoring/languages/azerbaijani.py

```diff
@@ -0,0 +1,27 @@
+from nltk.corpus import stopwords
+
+from .language import Language, LanguageUtility
+
+# Notice:
+# It seems like Azerbaijani stemmer isn't in the nltk snowball library for some reason:
+# http://www.nltk.org/_modules/nltk/stem/snowball.html
+# STEMMER = SnowballStemmer("azerbaijani")
+
+#STOPWORDS = set([])
+BADWORDS = set([])
+
+def is_badword_process():

```

Comment:
We shouldn't provide this function if it won't work.  It's much better to leave it a missing dependency.